charts/hydra/values.yaml

```diff
@@ -83,6 +83,21 @@ env:
   - name: LOG_LEVEL
     value: info
 
+containerSpec:
+  lifecycle:
+    enabled: true
+    preStop:
+      exec:
+        command: ["/bin/sh", "-c", "sleep 10"]
```

Comment:
This is not required to be enabled in default values, as this chart is being used by other apps as well. Let's keep this disabled and all values commented here.

Reply:
sure. disabled and commented out the preStop.